### ISSUE TITLE
Error during setup (Docker for Windows/Windows 10)

### ISSUE THREAD

**Issue by gaetoleole** [hoelsner/product-database#6]
Hi Henry,

I found your project when looking for eos info and it looks great !
I'm willing to test it but got the below error message when trying to setup the demo.

PS C:\product-database> docker-compose -p productdb up build_deps
Starting productdb_build_deps ...
Starting productdb_build_deps ... done
Attaching to productdb_build_deps
build_deps_1  | /usr/bin/env: 'bash\r': No such file or directory
productdb_build_deps exited with code 127

Any idea what the issue could be?
I'm running docker on windows 10.

Tx

**Comment by hoelsner** [comment]
Hi @gaetoleole,

I think there is an issue with the line-endings when cloning the source on Windows. Can you please check the autocrlf option in your git configuration (see https://help.github.com/articles/dealing-with-line-endings)? The line endings must only be LF and should not automatically converted to CRLF.

Greetings, Henry

**Comment by gaetoleole** [comment]
Hi Henry,
Thanks, it solved the issue.
Below the GIT config that I used.
PS C:\product-database> git config --global core.eol
lf
PS C:\product-database> git config --global core.autocrlf
input

Rgds
Gaet

**Comment by hoelsner** [comment]
Hi @gaetoleole,

I tested it today on Windows 10 running Docker for Windows Version 17.09.0 (CE) and the regular command-line interface (cmd.exe, not the PowerShell). I was able to reproduce the issue regarding the line endings within the build helper container. I fixed it by applying the following git configuration option: 

```
git config --global core.autocrlf false
```

After then, I cloned the repository again to make sure that the line endings are as expected and rebuild the containers according the [Setup & Installation Guide](https://github.com/hoelsner/product-database/blob/master/SETUP.md). It was working in my environment.

I see that you closed the issue and maybe you already fixed the issue on your system. I will add a note to the setup guide to prevent this issues in the future.

Kind regards, 
Henry
